Tests/UnitTests/Security/SigningTests.swift

```diff
@@ -276,6 +295,15 @@ class SigningTests: TestCase {
         )) == true
     }
 
+    /*
```

Comment:
@aboedo I added this for future reference :)
(This PR already has the correct signatures since @bisho got all the changes in the canary 🎉)